### ISSUE TITLE
https://ourboard.io/ domain loads forever and does not redirect.

### ISSUE THREAD

**Issue by rrsai** [raimohanska/ourboard#189]
# Feature request story or issue

## Screenshot

YOUR SCREENSHOT HERE:
![image](https://user-images.githubusercontent.com/56273094/146430641-3580e362-b2c2-48ee-9dea-eed1bf675dba.png)

## Url

-   Url where you took the screenshot:
-   URL: https://ourboard.io/

## A sentence description of the problem

Short DESCRIPTION:
going to the url https://ourboard.io/ specifically, so the https version of the domain without a www, does not appear to redirect or load and just spins and tries to load forever.

That's it! Thanks.

**Comment by raimohanska** [comment]
Fixed! 

I had the Route53 A record for ourboard.io pointing to a S3 hosted website that in turn is configured to redirect to https://www.ourboard.io.

The problem was that S3 doesn't support HTTPS website hosting. I also had to create a CloudFront distribution, which in turn requires a Certificate, which is validated using DNS (Route53 again). The CloudFront distribution allowed me to redirect HTTP traffic to HTTPS and finally the whole shabang works.

I cannot recommend using AWS for managing the domains for Heroku applications. I small part of my soul died during this process.
